### PR TITLE
[SYM-3635] Custom Strategy Quickstart

### PR DESCRIPTION
**Pull request by Arianna2028**
This PR contains the contents required to get up and running with a custom access strategy. It should be apply-able and then immediately usable, though it doesn't actually do any real escalation.

Screen Shot 2022-05-03 at 3 53 13 PM

### CODE REVIEW

**Comment by Arianna2028** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1** <a href="https://app.graphite.dev/github/pr/symopsio/sym-custom-strategy-quickstart/1" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/symopsio/sym-custom-strategy-quickstart/1?utm_source=stack-comment).

File: app/variables.tf [hunk 1]
```diff
@@ -0,0 +1,52 @@
+variable "aws_region" {
+  description = "AWS Region"
+  type        = string
+  default     = "us-east-1"
+}
+
+variable "error_channel" {
+  description = "The error channel to use to surface Sym errors."
```
Comment (sym-andrewa) [inline app/variables.tf]: Nit: This wording seems a bit repetitive; perhaps something like "The Slack channel to use to surface Sym errors" might be better? (Also has the benefit of clarifying what's meant by "channel".)

File: modules/custom-access-flow/main.tf [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+# A custom Integration can be used to access secrets in your custom strategy implementation
+# as well as manage identities.
+resource "sym_integration" "custom" {
+  type        = "custom"
+  name        = local.flow_name
+  external_id = var.integration_identifier
+
+  settings = {
+    api_token_secret = sym_secret.api_key.id
```
Comment (sym-andrewa) [inline modules/custom-access-flow/main.tf]:
```suggestion
    secret_ids_json = jsonencode([sym_secret.api_key.id])
```

File: modules/sym-runtime/variables.tf [hunk 3]
```diff
@@ -0,0 +1,34 @@
+variable "error_channel" {
+  description = "The error channel to use to surface Sym errors."
```
Comment (sym-andrewa) [inline modules/sym-runtime/variables.tf]: Nit: Same here with respect to the wording of the description

**Review by sym-andrewa — CHANGES_REQUESTED** [review]
Noticed an issue with the strategy settings, plus a wording nitpick

**Review by sym-andrewa — APPROVED** [review]
yee

**Pull request merged**